src/main/java/util/Convertable.java

```diff
@@ -0,0 +1,34 @@
+package util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONObject;
+
+public interface Convertable {
+
+    default Object getObject(JSONObject jsonObject, Class aClass) {
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        try {
+            return objectMapper.readValue(jsonObject.toString(), aClass);
+        } catch (JsonProcessingException e) {
+            System.out.println("Convertable error: " + e.getMessage());
+        }
+        return null;
+    }
+
+    default Object getObject(String jsonObject, Class aClass) {
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        try {
+            return objectMapper.readValue(jsonObject, aClass);
+        } catch (JsonProcessingException e) {
+            System.out.println("Convertable error: " + e.getMessage());
+        }
+        return null;
+    }
+
```

Comment:
It is very high abstraction to return Object as type, you need to manipulate your POJO classes. Or at least Generic if you know how to implement it.

Reply:
WIP

Reply:
Fixed